### PR TITLE
Adding 2022 CF workshop

### PR DESCRIPTION
**Pull request by cofinoa**

### CODE REVIEW

File: Meetings/2022-Workshop.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+The Climate and Forecast metadata conventions (CF) are a community-developed standard  designed to promote the processing and sharing of climate and forecast model and observational output data, and derived data products. The conventions define metadata that provide a definitive description of what the data in each variable represents, and the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities. The CF convention includes a standard name table, which defines strings that identify physical quantities.
+
+2022 CF Workshop will take place on 13-15 September 2022 hosted by [IFCA] at Santander (Spain). After 2 successful virtual editions ([2020] and [2021]), the 2022 edition is organized as a hybrid event having in-person and on-line participants. 
+
+[2021]: 2021-Workshop.html "2021 CF Workshop"
+[2020]: 2020-Workshop.html "2020 CF Workshop"
+
+## Registration
+
+There are no registration fees, but registration before 30th of August it's required.
```
Comment (davidhassell) [inline Meetings/2022-Workshop.md:17]:
```suggestion
There are no registration fees, but registration before 30th of August is required.
```

File: Meetings/2022-Workshop.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**TO BE COMPLETED**
+
+### Training session
+In the week prior to the CF Workshop we plan to run a virtual training event for users new to CF.
+This will take place on Thursday 8th September, 1500 - 1800 UTC. 
+
+**The full programme will be published on the CF website prior to the event.** 
+
+Pre-registration is not essential, but to help us with planning please the corresponding checkbox, in the [registration form][REGISTRATION], if you would be interested in attending. Videos of the training presentations will be made available for later viewing by those unable to attend the event itself.
```
Comment (davidhassell) [inline Meetings/2022-Workshop.md:44]:
```suggestion
Pre-registration for the on-line training event is not essential, but to help us with planning please the corresponding checkbox, in the [registration form][REGISTRATION], if you would be interested in attending. Videos of the training presentations will be made available for later viewing by those unable to attend the event itself. 
```

File: Meetings/2022-Workshop.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+### Topics for hackatons sessions
+
+We will devote the afternoon (CEST time) sessions of each day to focused-attention hackathons, during which it is hoped that concrete progress on each topic can be made (such as actual changes suggested to the Conventions text). 
+
+Please express an interest in any number of these sessions, in the [registration form][REGISTRATION], or suggest a new topic that could be included, with a suggestion of who you think could lead the discussion. 
+
+More details at: <https://github.com/cf-convention/discuss/issues/152>
+
+### Poster contributions 
+During the CF Workshop there will be available displays for Poster contributions from participants. There will not formal poster session but it's expected to informal chats over during breaks.
```
Comment (davidhassell) [inline Meetings/2022-Workshop.md:57]:
```suggestion
During the CF Workshop there will be available displays for Poster contributions from participants. There will be no formal poster session, but posters expected to chatted about informally during breaks.
```

**Review by davidhassell — COMMENTED** [review]
Hi @cofinoa - looks really good. Just a few text suggestions ...

File: Meetings/2022-Workshop.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+title: 2022 CF Workshop
+---
+
+# 2022 CF Workshop
+
+The Climate and Forecast metadata conventions (CF) are a community-developed standard  designed to promote the processing and sharing of climate and forecast model and observational output data, and derived data products. The conventions define metadata that provide a definitive description of what the data in each variable represents, and the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities. The CF convention includes a standard name table, which defines strings that identify physical quantities.
+
+2022 CF Workshop will take place on 13-15 September 2022 hosted by [IFCA] at Santander (Spain). After 2 successful virtual editions ([2020] and [2021]), the 2022 edition is organized as a hybrid event having in-person and on-line participants. 
+
+[2021]: 2021-Workshop.html "2021 CF Workshop"
```
[the remark this answers is not in the document]
Comment (cofinoa) [reply Meetings/2022-Workshop.md:12]: Check also the links at the `index.md` page in the `Meetings` dir.

File: Meetings/2022-Workshop.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+[2020]: 2020-Workshop.html "2020 CF Workshop"
```
[the remark this answers is not in the document]
Comment (cofinoa) [reply Meetings/2022-Workshop.md:13]: Check also the links at the index.md page in the Meetings dir.

**Comment by cofinoa** [comment]
@davidhassell 
> The old meeting links 404-ed for me.

I have accepted the absolute URLs for the old workshops, but the relative one should be OK. 

The relative links are been used in the `index.md` for the `Meetings` directory, and they are working.

**Pull request merged**